docs/use-cases/set-required-approvals/example.yml

```
@@ -0,0 +1,14 @@
+api-version: reviewpad.com/v3.x
+
+rules:
+  - name: changes-critical-code
+    kind: patch
+    description: Changes to critical code
+    spec: '$hasAnnotation("critical")'
+
+workflows:
+  - name: has-required-approvals
+    if:
+      - '$rule("changes-critical-code") && !$hasRequiredApprovals(2, $team("core"))'
+    then:
+      - '$fail("Blocking the merge")'
```

Can you please make sure that this reviewpad configuration is valid and works properly?

```suggestion
api-version: reviewpad.com/v3.x

rules:
  - name: critical-changes
    spec: '$hasAnnotation("critical")'

workflows:
  - name: has-required-approvals
    if:
      - '$rule("critical-changes") && !$hasRequiredApprovals(2, $team("core"))'
    then:
      - '$fail($sprintf("A minimum of 2 approvals ir requred from @%v", $team("core"))'
```